Source/WebKit/Shared/RemoteLayerTree/CGDisplayListImageBufferBackend.mm

```diff
@@ -79,9 +79,10 @@ void setCTM(const WebCore::AffineTransform& transform) final
         return m_immutableBaseTransform * GraphicsContextCG::getCTM(includeDeviceScale);
     }
 
-    WebCore::FloatRect roundToDevicePixels(const WebCore::FloatRect& rect, RoundingMode = RoundAllSides) const final
+    std::optional<std::pair<float, float>> scaleForRoundingToDevicePixels() const final
```

Comment:
I feel like we generally use `FloatSize` to represent scales in two dimensions – lots of results if you search for `FloatSize scale`, and very little to none for `std::pair<float, float>` / `std::pair<double, double>`.

Reply:
Oop, I missed this comment (collapsed on mobile GitHub by default?). That is a fair point (I just don’t love “width” and “height” for scales, but first and second isn’t great either).